qa/src/test/java/io/camunda/migrator/qa/history/HistoryTaskMigrationTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migrator.qa.history;
+
+import static io.camunda.search.entities.FlowNodeInstanceEntity.FlowNodeType.END_EVENT;
+import static io.camunda.search.entities.FlowNodeInstanceEntity.FlowNodeType.START_EVENT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.search.entities.ProcessInstanceEntity;
+import io.camunda.search.entities.UserTaskEntity;
+import java.util.List;
+import org.camunda.bpm.engine.task.Task;
+import org.junit.jupiter.api.Test;
+
+public class HistoryTaskMigrationTest extends HistoryMigrationAbstractTest {
+
+  @Test
+  public void shouldHistoricUserTasksBeMigrated() {
+    deployer.deployCamunda7Process("io/camunda/migrator/bpmn/c7/userTaskProcess.bpmn");
```

Comment:
💭 There's no C8 deployment here. If want to do a deployment, we will need the C8 client that will slow down the test execution.
My concern is: if we don't deploy the C8 process, will the data be consistent on the C8 side?
I think the answer is the C8 deployments are not like C7 deployment; there's no Deployment table and entity on C8 side. So in theory, it should work out-of-the-box.
Further, we will not do validation as we do for the Runtime.